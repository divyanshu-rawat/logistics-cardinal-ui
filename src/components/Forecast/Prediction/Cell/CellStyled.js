import styled from 'styled-components';

export default styled.div`
  overflow: hidden;
  width: 100%;
  min-height: 40px;
  position: relative;
  background-color: ${({ type, theme }) =>
    type === 'forecast' ? theme.colors.grey400 : theme.colors.grey300};
`;
