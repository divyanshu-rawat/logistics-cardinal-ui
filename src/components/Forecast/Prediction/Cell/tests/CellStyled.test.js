import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import CellStyled from '../CellStyled';
import Theme from '../../../../../themes/rooster';

const renderedComponent = (type = 'forecast') =>
  shallow(<CellStyled theme={Theme} type={type} />);

describe('<CellStyled />', () => {
  it('should render the styled component as `forecast`', () => {
    const tree = toJson(renderedComponent());
    expect(tree).toMatchSnapshot();
  });

  it('should render the styled component as `actual`', () => {
    const tree = toJson(renderedComponent('actual'));
    expect(tree).toMatchSnapshot();
  });
});
