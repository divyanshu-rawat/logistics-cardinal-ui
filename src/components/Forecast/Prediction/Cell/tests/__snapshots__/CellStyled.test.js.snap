// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CellStyled /> should render the styled component as \`actual\` 1`] = `
<div
  className="sc-bdVaJa kCuAvX"
  type="actual"
/>
`;

exports[`<CellStyled /> should render the styled component as \`forecast\` 1`] = `
<div
  className="sc-bdVaJa twrwa"
  type="forecast"
/>
`;
